package.json

```diff
@@ -1,9 +1,19 @@
 {
   "devDependencies": {
-    "babel-eslint": "^10.1.0",
```

Comment:
this was deprecated, so I removed and updated, hence all the files that got updated